src/Utils/utils.ts

```diff
@@ -544,3 +544,58 @@ export const fahrenheitToCelsius = (fahrenheit: number) => {
 export const keysOf = <T extends object>(obj: T) => {
   return Object.keys(obj) as (keyof T)[];
 };
+
+// Capitalizes the first letter of each word in a string, handling edge cases
+export const startCase = (str: string): string => {
+  if (!str || str.length === 0) return "";
+  return str
+    .replace(/([a-z])([A-Z])/g, "$1 $2")
+    .replace(/[_-]+/g, " ")
```

Comment:
Can you tell me where do we have usecases for replacing `_-` to spaces?

Also the half of this function seems unnecessary(The last 3 functions to be exact), considering that at the time of rendering `capitalize` can be used as a className

Reply:
sometimes maybe in 
user_name - user name
sample-test - sample test

Reply:
> sometimes maybe in user_name - user name sample-test - sample test

yes the last 3 functions do the same work as capitalize, but these are written in a way like
if we use startCase, we dont have to again use capitalize in the className

Reply:
@coderabbitai give more examples for this in the current codebase context

Reply:
Sir but the aim is to write equivalent functions of lodash
and separating will again involve writing span, capitalze and then again cypress debugging

Reply:
@rithviknishad 
startCase in lodash replaces _ , . - spaces and almost all other symbols in the default working
https://www.geeksforgeeks.org/lodash-_-startcase-method/